Package.swift

```diff
@@ -18,7 +18,7 @@ import PackageDescription
 let package = Package(
     name: "swift-openapi-urlsession",
     platforms: [
-        .macOS(.v13), .iOS(.v16), .tvOS(.v16), .watchOS(.v9),
+        .macOS(.v11), .iOS(.v13), .tvOS(.v13), .watchOS(.v6),
```

Comment:
For consistency, did you try bumping macOS back to `10.15`, so that all the OS's are aligned?

Reply:
Yes, sorry about that, missed to commit it.